tests/e2e/utils/PreferencesHandler.ts

```diff
@@ -15,19 +15,32 @@ export class PreferencesHandler {
      */
     public async setTerminalType(type: string) {
         Logger.debug('PreferencesHandler.setTerminalToDom');
+        await this.setPreference('terminal.integrated.rendererType', type);
+    }
+
+    /**
+     *
+     * @param askForConfirmation possible values are "never", "ifRequired" and "always"
+     */
+    public async setConfirmExit(askForConfirmation: string) {
+        Logger.debug(`PreferencesHandler.setConfirmExit to ${askForConfirmation}`);
+        await this.setPreference(`application.confirmExit`, askForConfirmation);
+    }
+
+    private async setPreference(attribute: string, value: string) {
+        Logger.debug(`PreferencesHandler.setPreferences ${attribute} to ${value}`);
         const response = await this.requestHandler.get('api/preferences');
         let userPref = response.data;
         try {
             let theiaPref = JSON.parse(userPref['theia-user-preferences']);
```

Comment:
A small improvement, everywhere in this method "let" may be changed to "const" and according to best practices better to use "const" everywhere it possible instead "let" for ensuring object immutability.

```
// doesn't work
const example = "test"
example = "another test"


// works
const example = "test"
example.field = "another test"

// works
const example = "test"
example['field'] = "another test"

Reply:
fixed